style.css

```diff
@@ -0,0 +1,123 @@
+@import url("https://fonts.googleapis.com/css?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap");
+
+body {
+  background-color: white;
+  font-family: "Poppins", sans-serif;
+  margin: 0;
+  padding: 0;
+}
```

Comment:
1. [ ] You have done a great job trying to implement the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=75%3A35) design. However, there are some aspects I would like you to consider.
    - Please use the background Images used in the Figma design. 
        Your design does not have background images.
    - Please try to center your elements. Your elements are currently 
       squeezed to the right.  You can use Margins.

### your design
![image](https://user-images.githubusercontent.com/98453979/185218309-e39f868e-2136-4bd2-8e23-956d0e488326.png)

### Figma design
![image](https://user-images.githubusercontent.com/98453979/185218441-39780e76-ecad-4a7e-8e3e-21d6ebbe1b03.png)